concepticondata/conceptlists/Koelle-1854-280.tsv

```diff
@@ -0,0 +1,322 @@
+Koelle-1854-280-50	50	Tooth	Tooth	1380	TOOTH
+Koelle-1854-280-51	51	Tongue	Tongue	1205	TONGUE
+Koelle-1854-280-52	52	Throat	Throat	1346	THROAT
+Koelle-1854-280-53	53	Gullet	Gullet		
+Koelle-1854-280-54	54	Neck	Neck	1333	NECK
+Koelle-1854-280-55	55	Shoulder	Shoulder	1482	SHOULDER
+Koelle-1854-280-56	56	Arm	Arm	1673	ARM
+Koelle-1854-280-57	57	Arm between Shoulder and Elbow	Arm Between Shoulder and Elbow	431	UPPER ARM
+Koelle-1854-280-58	58	Arm between Elbow and Wrist	Arm Between Elbow and Wrist	446	LOWER ARM
+Koelle-1854-280-59	59	Leg	Leg	1297	LEG
+Koelle-1854-280-60	60	Outer Hand, or Hand	Outer Hand, or Hand	1277	HAND
+Koelle-1854-280-61	61	Inner Hand	Inner Hand	1183	PALM OF HAND
+Koelle-1854-280-62	62	Foot, or Instep of the Foot	Foot, or Instep of the Foot	1301	FOOT
+Koelle-1854-280-63	63	Foot-sole	Foot-Sole	2666	SOLE (FOOT)
+Koelle-1854-280-64	64	Finger	Finger	1303	FINGER
+Koelle-1854-280-64a	64a	thumb	Finger (thumb)	1781	THUMB
+Koelle-1854-280-65	65	Toe	Toe	1389	TOE
+Koelle-1854-280-65a	65a	the big one [big toe]	Toe (the big one)		
+Koelle-1854-280-66	66	Elbow	Elbow	981	ELBOW
+Koelle-1854-280-67	67	Rib	Rib	801	RIB
+Koelle-1854-280-68	68	Chest	Chest	1592	CHEST
+Koelle-1854-280-69	69	Female Breast	Female Breast	1402	BREAST
+Koelle-1854-280-70	70	Belly	Belly	1251	BELLY
+Koelle-1854-280-71	71	Navel	Navel	1838	NAVEL
+Koelle-1854-280-72	72	Thigh	Thigh	471	UPPER LEG (THIGH)
+Koelle-1854-280-73	73	Knee	Knee	1371	KNEE
+Koelle-1854-280-74	74	Heel	Heel	980	HEEL
+Koelle-1854-280-75	75	Nail (of finger & toe)	Nail (of Finger and Toe)	1896	FINGERNAIL OR TOENAIL
+Koelle-1854-280-75a	75a	Nail (of fingers)	Nail (of Finger)	1258	FINGERNAIL
+Koelle-1854-280-76	76	Skin		763	SKIN
+Koelle-1854-280-76b	76b	Skin (body)	body		
+Koelle-1854-280-77	77	Bone	Bone	1394	BONE
+Koelle-1854-280-78	78	Vein	Vein	1924	VEIN
+Koelle-1854-280-78a	78a	rope	Vein (rope)	1218	ROPE
+Koelle-1854-280-79	79	Blood	Blood	946	BLOOD
+Koelle-1854-280-80	80	Itch	Itch	148	ITCH
+Koelle-1854-280-81	81	Small-pox	Small-Pox	1054	SMALLPOX
+Koelle-1854-280-82	82	Hat	Hat	771	HAT
+Koelle-1854-280-83	83	Cap	Cap	1288	CAP
+Koelle-1854-280-84	84	Shoe	Shoe	1381	SHOE
+Koelle-1854-280-85	85	Shirt	Shirt	1622	SHIRT
+Koelle-1854-280-86	86	Trousers	Trousers	809	TROUSERS
+Koelle-1854-280-87	87	Waist-cloth	Waist-cloth		
+Koelle-1854-280-88	88	Town (village)	a walled town//Village//Town (Village)//town (walled)//town	1391	TOWN
+Koelle-1854-280-89	89	Market		633	MARKET
+Koelle-1854-280-90	90	House		1252	HOUSE
+Koelle-1854-280-91	91	Door	Door (of grass)//Door (of wood)	1567	DOOR
+Koelle-1854-280-92	92	Door-way			
+Koelle-1854-280-93	93	bed	Bed	1663	BED
+Koelle-1854-280-94	94	mat	Mat	195	MAT
+Koelle-1854-280-95	95	Knife	Knife	1352	KNIFE
+Koelle-1854-280-96	96	Spoon		1378	SPOON
+Koelle-1854-280-96a	96a	Spoon (of wood)	Spoon (of wood)		
+Koelle-1854-280-97	97	Ear-ring	Ear-Ring	770	EARRING
+Koelle-1854-280-97a	97a	nose-ring	Nose-Ring		
+Koelle-1854-280-98	98	Armlet or Bracelet	Armlet or Bracelet		
```

Comment:
ARMLET

Reply:
Or wait, seems like an armlet and a bracelet are not synonyms, bracelets are for the wrist, armlets/arm rings for the upper arm). Perhaps we want to edit the definition of ARMLET?
("A band or bracelet worn round the upper part of a person's arm."
-> "A band of metal worn round the upper part of a person's arm.")

Reply:
Yes, sounds great, I think it would be nice to keep ARMLET and BRACELET apart for the time being as this list is the only one with the merge concept.